generators/rust/codegen/src/custom-config/BaseRustConfigSchema.ts

```diff
@@ -13,7 +13,9 @@ export const BaseRustCustomConfigSchema = z.object({
     packageDescription: z.string().optional(),
     packageLicense: z.string().optional(),
     packageRepository: z.string().optional(),
-    packageDocumentation: z.string().optional()
+    packageDocumentation: z.string().optional(),
+    // CI/CD configuration
+    enablePublishWorkflow: z.boolean().optional().default(false)
```

Comment:
Why not default to true? I think we do that in other languages.

also can you make sure this aligns with whatever flags we use in other languages